src/garage/_dtypes.py

```diff
@@ -167,3 +167,14 @@ def __new__(cls, env_spec, observations, actions, rewards, terminals,
         return super().__new__(TrajectoryBatch, env_spec, observations,
                                actions, rewards, terminals, env_infos,
                                agent_infos, lengths)
+
+class TorchTrajectoryBatch(
```

Comment:
wait, how does this differ other than having no validation?

Reply:
differ from TrajectoryBatch, that is

Reply:
is it to add baselines?

The only point in adding a new type is to define an interface between two modules. `TrajectoryBatch` defines the data interface between samplers and algorithms. `baselines` is internal to policy gradient algorithms.

I'm not sure what structure makes sense here, but you should first try just passing baselines as its own argument to the functions you need. If that doesn't work, I suggest you define a new dtype, perhaps in torch/VPG.

Reply:
also, whatever you do with this, please break it into a separate PR

Reply:
I think the idea is that this contains `torch.FloatTensor` instead of `np.ndarray`. It seems like a reasonable thing to have, but probably belongs in `garage.torch`.